### PR TITLE
Dashboard Page

### PR DESCRIPTION
**Pull request by imsamtar**
Here is what is done so far

1. Setup sveltekit with typescript (and scss because it is requited by svelte-materialify).
2. Add sveltemeterialify, chart.js
3. Add first pages dashboard landing page

### CODE REVIEW

File: svelte.config.cjs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+/** @type {import('@sveltejs/kit').Config} */
+module.exports = {
+	// Consult https://github.com/sveltejs/svelte-preprocess
+	// for more information about preprocessors
+	preprocess: sveltePreprocess(),
+	kit: {
+		// By default, `npm run build` will create a standard Node app.
+		// You can create optimized builds for different platforms by
+		// specifying a different adapter
+		adapter: node(),
```
Comment (jozef-slezak) [inline svelte.config.cjs:14]: please use `adapter: static(),` if possible

File: src/lib/Drawer.svelte [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+				Customers
+			</ListItem>
+		</List>
+		<span slot="append" class="pa-2">
+			<Button block>Logout</Button>
+		</span>
+	</NavigationDrawer>
+</div>
+
+<style lang="scss">
```
Comment (jozef-slezak) [inline src/lib/Drawer.svelte]: lets not use scss 
lets use css only (you still have css variables)
but if you see the need of scss please point that out

File: src/lib/dashboard/TopCards.svelte [hunk 3]
```diff
@@ -0,0 +1,66 @@
+<script>
+	import { Avatar, Card, CardActions, ProgressLinear } from 'svelte-materialify';
+	import { topCards } from './data';
+</script>
+
+{#each topCards as card}
```
Comment (jozef-slezak) [inline src/lib/dashboard/TopCards.svelte]: Please avoid this loop/each. I would like dedicated component for each card (same as example-material-ui)

**Comment by jozef-slezak** [comment]
@imsamtar, thank you for the first pull request. At the moment I am finished with the code review. After you resolve these review comments I am going to configure automatic deployment / preview of pull requests for quick visual checks.

**Comment by imsamtar** [comment]
Creating new pull request

**Comment by imsamtar** [comment]
#3 is new pull request

**Pull request closed without merging**